github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml

```diff
@@ -427,11 +427,13 @@ external_plugins:
     events:
       - pull_request
   - name: cherrypicker
+  - name: release-blocker
```

Comment:
Do we need to build a new image and update the deployment manifest before adding this back in? 

https://github.com/kubevirt/project-infra/blob/f96e43d0b8d6aec90ef48b8b02b49b3266d9be74/github/ci/prow-deploy/kustom/base/manifests/local/release-blocker_deployment.yaml#L20